doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -1005,7 +1006,30 @@
     std::string myConvertedIoxString = iox::into<std::string>(myIoxString);
     ```
 
-44. Move and rename `DeadlineTimer`
+44. In order to use the comparison or `operator<<` operators, `insert`, `find`, `unsafe_append` functions for
+    `std::string` together with `iox::string` include the support header:
+
+    ```cpp
+    // before
+    std::string myStdString("foo");
+    iox::string<3> myIoxString(TruncateToCapacity, myStdString);
+    if(myIoxString == myStdString)
+    {
+      ..
+    }
+
+    // after
+    #include "iceoryx_dust/cxx/std_string_support.hpp"
+
+    std::string myStdString("foo");
+    iox::string<3> myIoxString(TruncateToCapacity, myStdString);
```

Comment:
This should be now something like `iox::string<3> myIoxString(into<lossy<iox::string<3>>>(myStdString);`

Reply:
@FerdinandSpitzschnueffler Good catch, fixed.